artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/CoreAmqpConverter.java

```diff
@@ -95,7 +95,7 @@
 public class CoreAmqpConverter {
 
    private static Logger logger = Logger.getLogger(CoreAmqpConverter.class);
-
+   public static final String INTERNAL_HEADER = "__HDR_";
```

Comment:
I am thinking whether this variable should have a broader scope and be consistently used instead of the string constant. I know of at least one other such place, the `__HDR_` stripping code in https://github.com/apache/activemq-artemis/pull/1320/files.

Reply:
This should be within the OpenWire protocol module and code, should avoid protocols specifics leaking into other protocol modules / code area's

Reply:
+1